### PR TITLE
Dense note paragraph --> H4 section

### PR DESCRIPTION
**Pull request by thedriftofwords**
My attempt at parsing the note might not be fully accurate.

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks, Crystal! Looping in Priya to confirm

File: site/en/r2/tutorials/distribute/training_loops.ipynb [hunk 1]
```diff
@@ -702,7 +702,22 @@
...
+        "### Tracking training loss across replicas\n",
+        "\n",
+        "As a general rule, you should use `tf.keras.Metrics` to track per-sample values and avoid values that have been aggregated within a replica. Loss scaling with `tf.keras.Metrics` accounts for the number of replicas in sync.\n",
```
Comment (yashk2810) [inline site/en/r2/tutorials/distribute/training_loops.ipynb]: Maybe put this in a "Note: As a general rule, you should use `tf.keras.Metrics` to track per-sample values and avoid values that have been aggregated within a replica. Loss scaling with `tf.keras.Metrics` accounts for the number of replicas in sync.". Rest of it can be outside the note.

**Comment by guptapriya** [comment]
@anj-s can you review this change as your recently added this description? thanks

File: site/en/r2/tutorials/distribute/training_loops.ipynb (hunk 1 above)
Comment (anj-s) [inline site/en/r2/tutorials/distribute/training_loops.ipynb]: we can remove this line "Loss scaling with `tf.keras.Metrics` accounts for the number of replicas in sync" since it makes the note confusing to the reader.

File: site/en/r2/tutorials/distribute/training_loops.ipynb [hunk 2]
```diff
@@ -702,7 +702,22 @@
...
+        "\n",
+        "We do *not* recommend using `tf.metrics.Mean` to track the training loss across different replicas, because it does not account for the number of replicas in\n",
```
Comment (anj-s) [inline site/en/r2/tutorials/distribute/training_loops.ipynb]: "because it does not account for the number of replicas" can be replaced with "because of the loss scaling computation that we carry out" or something to that effect.

File: site/en/r2/tutorials/distribute/training_loops.ipynb [hunk 3]
```diff
@@ -702,7 +702,22 @@
...
+        "sync.\n",
+        "\n",
+        "For example, if you run a training job with the following characteristics:\n",
+        "* Two replicas\n",
+        "* Two samples are processed on each replica\n",
+        "* Resulting loss values: [2,  3] and [4,  5] on each replica\n",
+        "* Global batch size = 4\n",
+        "\n",
+        "Loss calculated with `tf.keras.Metrics` is scaled by an additional factor that is equal to the number of replicas in sync. With loss scaling, you calculate the per-sample value of loss on each replica by adding the loss values, and then dividing by the global batch size. In this case: `(2 + 3) / 4 = 1.25` and `(4 + 5) / 4 = 2.25`. \n",
```
Comment (anj-s) [inline site/en/r2/tutorials/distribute/training_loops.ipynb]: It makes more sense for "Loss calculated with `tf.keras.Metrics` is scaled by an additional factor that is equal to the number of replicas in sync." to be the last line in this section.

**Comment by yashk2810** [comment]
I am pulling this in and will make the changes, since this seems like an important change for readability.

**Comment by thedriftofwords** [comment]
@yashk2810 Thanks for taking care of that!
